Sources/Converters/Internal/SubtitleTimestampFormatter.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-struct SubtitleTimestampFormatter {
```

Comment:
What was the problem with this being a `struct`?

Reply:
SwiftLint: https://realm.github.io/SwiftLint/convenience_type.html

Reply:
Oh, nice. 👍